data/snippets.conf

```diff
@@ -46,6 +53,7 @@ do=do\n{\n\t%cursor%\n} while (%cursor%)\n%cursor%
 switch=switch (%cursor%)%brace_open%case %cursor%:\n\t\t%cursor%\n\t\tbreak;\n\tdefault:\n\t\t%cursor%\n%brace_close%%cursor%
 
 [C++]
+main=\n\nint main(int argc, char* argv[]) {\n\t%cursor%\n\treturn 0;\n}
```

Comment:
Yes only one main, but just typing one "main" to have complete main is very fast.